conda/cli/install.py

```diff
@@ -212,13 +194,6 @@ def install(args, parser, command='install'):
             raise CondaValueError("cannot mix specifications with conda package"
                                   " filenames")
 
-    # handle tar file containing conda packages
-    if len(args.packages) == 1:
-        tar_path = args.packages[0]
-        if tar_path.endswith('.tar'):
```

Comment:
I think this is all dead code.  It's all completely untested at least by all of our integration tests.  I don't know what code path would actually get to it.  You'd need to `conda install my-package.tar` from the command line.  Does anybody do that?